src/views/Login/index.js

```diff
@@ -1 +1,2 @@
 export { default } from './Login';
+export { default as Logout } from './Logout';
```

Comment:
login and logout have same export name :  default. 
Is that ok?

Reply:
Here Login is exported as default and Logout is exported as Logout. 

the way you reference is  as follows:
`
import Login, { Logout } from './views/Login'/
`
I put Login and Logout views in same folder as they both relate to each other. Since the main view name is `Login`, I thought of exporting `Login` as default and `Logout` as sub module.

Reply:
good!